README.md

```diff
@@ -391,6 +392,44 @@ Setup `shibboleth` IdP:
 
 > Note: SLO is not fully configured, yet
 
+### Enabling HAProxy
+
+For testing Ceph Dashboard HA you should run multiple Ceph Managers. To do so
+execute the following command in your Ceph development environment:
+
+    # export MGR=3
+    # start-ceph.sh
+
+Now you have to update the HAProxy configuration file ./haproxy/haproxy.conf
+and adapt the host ports of your running Ceph Dashboards. After that start
```

Comment:
Is there an easy way to get these ports? Perhaps we should add a hint on how to get them.

Reply:
> Is there an easy way to get these ports? 
No, not that i know off. You need to extract the ports from the output of the vstart script. I will add a hint where to find and extract the ports.